internal/pkg/template/use_test.go

```diff
@@ -85,10 +85,10 @@ func TestUseContext(t *testing.T) {
   "Input3": 3.5,
   "Input4": false,
   "Objects": {
-    "Config1": "<<~~ticket:1~~>>",
-    "Config2": "<<~~ticket:1~~>>",
-    "Row1": "<<~~ticket:2~~>>",
-    "Row2": "<<~~ticket:2~~>>"
+    "Config1": "<<~~placeholder:1~~>>",
```

Comment:
Upravil som trocha placeholder, kedze sa nejedna vzdy o nove ID/ticket. Je to iba interna hodnota.